src/Store.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useReducer } from 'react';
+import { stateReducer } from './reducers/stateReducer';
+import { filters } from './lib/filters';
+import { Action } from './types/Action';
+import { State } from './types/State';
+
+const defaultDispatch: React.Dispatch<Action> = () => {};
+
+const initialState: State = {
+  todos: [],
+  filteredTodos: [],
```

Comment:
please fix prev comment.
it is redundant to make two states for todos.
rewrite logic wihtout filtered todos.
in all places where you need filtered todos - just filter todos by active filter.